src/modules/layout/components/HeaderNavigationItem.js

```diff
@@ -14,7 +14,7 @@ class HeaderNavigationItem extends React.PureComponent {
     /** text to be displayed */
     text: PropTypes.string.isRequired,
     /** link to the page that should be shown when the item is clicked */
-    href: PropTypes.string.isRequired,
+    href: PropTypes.object.isRequired,
```

Comment:
because redux-first-router links are using actions to link to different pages, see https://github.com/faceyspacey/redux-first-router-link. Should also be refactored to type: Action when flow is introduced here